vulns/idna/PYSEC-2024-60.yaml

```diff
@@ -23,7 +23,8 @@ affected:
     repo: https://github.com/kjd/idna
   - type: ECOSYSTEM
     events:
-    - introduced: "0"
```

Comment:
NB: Changed this from `0` since it's unclear where that came from; the first vulnerable version listed below is `0.1`.